assets/css/components/button.scss

```
@@ -0,0 +1,81 @@
+.btn {
+  @extend %font-ui-text-base-good;
+  padding: 0 10px;
+  color: $color-white-160;
+  background-color: transparent;
+  border-style: none;
+  border-radius: 0;
+  opacity: 0.7;
```

```suggestion
  opacity: 0.7;
  transition: all 0.3s ease-in-out;
```

The last thing this needs is a transition effect so that when the state of the button changes, it will smoothly transition the text, background, & border colors. Generally speaking, it's best to specify which properties need to be transitioned, but in the case of these buttons (when it can vary), it's acceptable to set it to `all`.